tests/sentry/api/endpoints/test_rule_snooze.py

```diff
@@ -81,22 +84,23 @@ def test_mute_issue_alert_everyone_forever(self):
         assert response.data["ruleId"] == self.issue_alert_rule.id
         assert response.data["alertRuleId"] is None
         assert response.data["until"] == "forever"
-        assert AuditLogEntry.objects.filter(
+        event = log_rpc_service.find_last_log(
```

Comment:
Open to keeping `AuditLogEntry` with an added `assume_test_silo_mode` too, I switched this one over since it was possible exactly match the existing assertion. Do we have a preference in tests?

Reply:
I think using `log_rpc_service` is fine. Having usage of these services in tests will help provide more examples of their usage to other teams and reduce the usage of escape hatches which seems like a win.